.gitmodules

```
@@ -1,3 +1,3 @@
 [submodule "seqan"]
 	path = include/seqan
-	url = git://github.com/seqan/seqan.git
+	url = https://github.com/seqan/seqan.git
```

```suggestion
	url = ../../seqan/seqan.git
```

While we are at it, we can just switch to relative syntax (this will preserve whatever protocol is used for the main repo).